fireblocks_sdk/sdk.py

```diff
@@ -1800,6 +1801,304 @@ def remove_web3_connection(self, session_id: str):
 
         return self._delete_request(url)
 
+    def get_smart_transfer_tickets(self, paged_smart_transfer_request_filters: GetSmartTransferFilters):
+        """Gets a page of smart transfer for your tenant according to filters given
+        Args:
+            paged_smart_transfer_request_filters (object, optional): Possible filters to apply for request
+        """
+
+        url = f"/v1/smart-transfers"
+
+        q, statuses, network_id, created_by_me, expires_after, expires_before, ticket_type, external_ref_id, limit,\
+            after = attrgetter('q', 'statuses', 'network_id', 'created_by_me', 'expires_after', 'expires_before',
+                               'ticket_type', 'external_ref_id', 'limit', 'after')(paged_smart_transfer_request_filters)
```

Comment:
why this method of getting attributes? 
why not just `paged_smart_transfer_request_filters.<field>` when needed?